src/trees/MultiResolutionAnalysis.cpp

```diff
@@ -77,20 +118,48 @@ MultiResolutionAnalysis<D>::MultiResolutionAnalysis(const BoundingBox<D> &bb, co
     setupFilter();
 }
 
+/** @returns Boolean value
+ *
+ *  @brief Equality operator for the MultiResolutionAnalysis class, returns true if both MRAs have the same polynomial basis, computational domain and maximum depth, and false otherwise
+ *
+ * @param[in] mra: MRA object, taken by constant reference
+ *
+ *  @details Equality operator for the MultiResolutionAnalysis class, returns true if both MRAs have the same polynomial basis represented by a BoundingBox object, computational domain (ScalingBasis object) and maximum depth (integer), and false otherwise.
+ *  Computations on different MRA cannot be combined, this operator can be used to make sure that the multiple MRAs are compatible.
+ *  For more information about the meaning of equality for BoundingBox and ScalingBasis objets, see their respective classes.
+ */
 template <int D> bool MultiResolutionAnalysis<D>::operator==(const MultiResolutionAnalysis<D> &mra) const {
     if (this->basis != mra.basis) return false;
     if (this->world != mra.world) return false;
     if (this->maxDepth != mra.maxDepth) return false;
     return true;
 }
 
+/** @returns Boolean value
+ *
+ *  @brief Inequality operator for the MultiResolutionAnalysis class, returns false if both MRAs have the same polynomial basis, computational domain and maximum depth, and true otherwise
+ *
+ * @param[in] mra: MRA object, taken by constant reference
+ *
+ *  @details Inequality operator for the MultiResolutionAnalysis class, returns true if both MRAs have the same polynomial basis represented by a BoundingBox object, computational domain (ScalingBasis object) and maximum depth (integer), and false otherwise.
+ *  Opposite of the == operator.
+ *  For more information about the meaning of equality for BoundingBox and ScalingBasis objets, see their respective classes.
+ */
 template <int D> bool MultiResolutionAnalysis<D>::operator!=(const MultiResolutionAnalysis<D> &mra) const {
     if (this->basis != mra.basis) return true;
     if (this->world != mra.world) return true;
     if (this->maxDepth != mra.maxDepth) return true;
     return false;
 }
 
+/** @returns void
```

Comment:
```suggestion
/**
```
this seems redundant.